include/psp2/apputil.h

```diff
@@ -158,6 +158,9 @@ int sceAppUtilShutdown();
 //! Receive app event
 int sceAppUtilReceiveAppEvent(SceAppUtilAppEventParam *eventParam);
 
+//! Parse received app event from LiveArea
+int sceAppUtilAppEventParseLiveArea(SceAppUtilAppEventParam *eventParam, char *buffer);
```

Comment:
Is const-ness correct here?

Reply:
[This](https://github.com/Rinnegatamante/vitaQuake/blob/82af45c89559e48491791c1368855f17e146ec57/source/sys_psp2.c#L502) seems to indicate it's not `const`.

Reply:
for SceAppUtilAppEventParam